### PR TITLE
hardkernel/odroid_h4: Add blobs

### PR DESCRIPTION
**Pull request by philipandag**

### CODE REVIEW

File: hardkernel/odroid_h4/README.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    Version: v1.0,
+    License: PV Intel OBL Software License Agreement 11.2.2017
+
+Intel Management Engine and Flash Descriptor have been generated from Intel
+CSME for software kit using genuine components provided by Intel, subject
+to the PV Intel OBL Software License Agreement 11.2.2017.
+
+You can verify the binary hashes using signatures made by the Dasharo release
+keys for given platform available on 3mdeb-secpack. Current signing key:
+[Dasharo release 0.9.x for hardkernel signing key](https://github.com/3mdeb/3mdeb-secpack/tree/master/customer-keys/hardkernel/odroid/dasharo-release-0.9.x-for-hardkernel-signing-key.asc)
```
[the remark this answers is not in the document]
Comment (miczyg1) [reply hardkernel/odroid_h4/README.md:26]: It is good enough for now. We will get them signed before the final release of the firmware.

File: hardkernel/odroid_h4/README.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+Hardkernel ODROID H4+ platforms.
+
+Do not use or load software from this repository until you have carefully
+read the licenses assigned to the relevant components. By downloading or
+using the software component from this repository, you agree to the terms
+of license associated with the given software component. If you do not wish
+to so agree, do not install or use the software from this repository.
+Association of the software components and licenses is presented below.
+
+- me.bin - Intel Management Engine
```
Comment (miczyg1) [inline hardkernel/odroid_h4/README.md:13]:
```suggestion
- `me.bin` - Intel Management Engine
```

File: hardkernel/odroid_h4/README.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+    Version: v16.50.10.1351,
+    License: PV Intel OBL Software License Agreement 11.2.2017
+- descriptor.bin - Intel Flash Descriptor
```
Comment (miczyg1) [inline hardkernel/odroid_h4/README.md:16]:
```suggestion
- `descriptor.bin` - Intel Flash Descriptor
```

File: hardkernel/odroid_h4/README.md [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+Do not use or load software from this repository until you have carefully
+read the licenses assigned to the relevant components. By downloading or
+using the software component from this repository, you agree to the terms
+of license associated with the given software component. If you do not wish
+to so agree, do not install or use the software from this repository.
+Association of the software components and licenses is presented below.
+
+- me.bin - Intel Management Engine
+    Version: v16.50.10.1351,
+    License: PV Intel OBL Software License Agreement 11.2.2017
```
Comment (miczyg1) [inline hardkernel/odroid_h4/README.md:15]: You didn't copy the template I have provided, did you?

In my template license was a link. Please fix.

File: hardkernel/odroid_h4/README.md [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+- descriptor.bin - Intel Flash Descriptor
+    Version: v1.0,
+    License: PV Intel OBL Software License Agreement 11.2.2017
```
Comment (miczyg1) [inline hardkernel/odroid_h4/README.md:18]: Same as above

File: hardkernel/odroid_h4/README.md [hunk 6]
```diff
@@ -0,0 +1,26 @@
+# Blobs
+
+The directory contains blobs to produce a full Dasharo firmware image for
+Hardkernel ODROID H4+ platforms.
```
Comment (miczyg1) [inline hardkernel/odroid_h4/README.md:4]:
```suggestion
Hardkernel ODROID H4 platforms.
```

File: hardkernel/odroid_h4/README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (miczyg1) [reply hardkernel/odroid_h4/README.md:26]:
```suggestion
[Dasharo release 0.9.x compatible with Hardkernel ODROID signing key](https://github.com/3mdeb/3mdeb-secpack/tree/master/customer-keys/hardkernel/odroid/dasharo-release-0.9.x-compatbile-with-hardkernel-odroid-signing-key.asc) 
```

The string will be `compatible with` as we do not have any agreement with Hardkernel on providing this firmware.

**Review by miczyg1 — CHANGES_REQUESTED** [review]

File: hardkernel/odroid_h4/README.md (hunk 4 above)
Comment (philipandag) [reply hardkernel/odroid_h4/README.md]: I did not. I have used a README for another platform instead, naively thinking that it's the same as what you wrote in the documentation, judging by a quick look.

https://github.com/Dasharo/dasharo-blobs/pull/23/commits/176bc1cb26cdefd1908c90fe6c5217fab58620a2#diff-708d08b771dcd3dbf87889870402c75472255110ace4191ddb8fb65e4f143feaR13-R18

File: hardkernel/odroid_h4/README.md (hunk 5 above)
Comment (philipandag) [reply hardkernel/odroid_h4/README.md]: https://github.com/Dasharo/dasharo-blobs/pull/23/commits/176bc1cb26cdefd1908c90fe6c5217fab58620a2#diff-708d08b771dcd3dbf87889870402c75472255110ace4191ddb8fb65e4f143feaR13-R18

File: hardkernel/odroid_h4/README.md (hunk 6 above)
Comment (miczyg1) [reply hardkernel/odroid_h4/README.md:4]: Not applied.

File: hardkernel/odroid_h4/README.md [hunk 7]
```diff
@@ -0,0 +1,35 @@
...
+
+* `me.bin` - Intel Management Engine
+  * Version: v16.50.10.1351,
+  * License:  [PV Intel OBL Software License Agreement 11.2.2017][INTEL SLA].
+* `descriptor.bin` - Intel Flash Descriptor
+  * Version: v1.0,
+  * License:  [PV Intel OBL Software License Agreement 11.2.2017][INTEL SLA].
+
+Intel Management Engine and Flash Descriptor have been generated from Intel
+CSME for Gracemont v16.50.10.1351 software kit using genuine components provided by Intel, subject to the
```
Comment (miczyg1) [inline hardkernel/odroid_h4/README.md:21]: Gracemont is a core architecture. This should be Alder Lake-N. Also this lien needs wrapping.

**Pull request merged**